app/models/episode_story_handler.rb

```diff
@@ -54,7 +54,7 @@ def update_attributes
     episode.summary = sa[:description]
     episode.content = sa[:description]
     episode.categories = sa[:tags]
-    episode.published_at = Time.parse(sa[:published_at]) if sa[:published_at]
+    episode.published_at = sa[:published_at] ? Time.parse(sa[:published_at]) : nil
```

Comment:
In CMS, "unpublish" is just a simple attribute update.  So yes, I think this is how we need to handle that here.